components/ContractViewer/ContractTreeView.tsx

```diff
@@ -0,0 +1,49 @@
+import ChevronRightIcon from '@mui/icons-material/ChevronRight'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import { TreeView } from '@mui/x-tree-view/TreeView'
+
+import { ContractInfo } from './ContractState'
+import ContractTreeNode from './ContractTreeNode'
+
+type ContractTreeViewProps = {
+  forest: any[]
+  onSelect: (node, root) => void
+}
+
+const ContractTreeView = ({ forest, onSelect }: ContractTreeViewProps) => {
+  // let expanded: string[] = []
+  // expanded = forest.map((t) => t.defTree.id as string)
```

Comment:
Charis mentioned this already but all commented out code should either be commented in again or removed before we can merge this.